### DISCUSSION TITLE
Model Viewer Animated Texture Question

### DISCUSSION THREAD

**Discussion by emincanyuksel** [google/model-viewer#4320] [General]
I have a question, I don't quite understand how to use the file here. Is there a sample file?
https://modelviewer.dev/examples/scenegraph/#animatedTexturesExample

**Comment by diegoteran** [comment]
Yes, the glb and the video are both files you should be able to access.
Video: https://modelviewer.dev/shared-assets/models/lottie-logo.mp4 (on the example it is loaded from "../../shared-assets/models/lottie-logo.mp4")
Model: https://modelviewer.dev/shared-assets/models/sphere.glb

If you change these paths to any available video/glb to your browser, you should be able to see other models/textures.
